src/ell/studio/server.py

```diff
@@ -31,15 +31,70 @@ def get_serializer(config: Config):
         raise ValueError("No storage configuration found")
 
 
+pubsub = None
```

Comment:
this is  initialized via the lifecycle handler and it looks like that doesn’t run in tests. So tests would need to mock it always. 

I’ll read fast api docs and try to figure something out.

Reply:
Normalized on fastapi depends